code/pizzabot.js

```diff
@@ -1,7 +1,53 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
 
 //Put your Javscript code here:
+
+//test
+//console.log(`Test if connection HTML-JS works: ${vegetarian}, ${hawaiian} and ${pepperoni}`);
+
+alert(`Welcome, customer! Happy to serve your pizza. On our menu we have: ${vegetarian}, ${hawaiian} and ${pepperoni}`);
+
+const orderName = prompt(`Enter the name of the pizza you want to order today:`);
+
+//Function that tests whether the user input matches the pizza name
```

Comment:
Great comment! When you're writing the other functions, you can start by making comments like these to clarify to yourselves what the function should do. E.g. for `calculateTotalCost()`
`//This function will calculate the total cost of the order by multiplying the orderQuantity and the pizzaPrice`